src/v/security/scram_authenticator.cc

```diff
@@ -21,7 +21,7 @@ result<bytes>
 scram_authenticator<T>::handle_client_first(bytes_view auth_bytes) {
     // request from client
     _client_first = std::make_unique<client_first_message>(auth_bytes);
-    vlog(seclog.debug, "Received client first message {}", *_client_first);
```

Comment:
Yeah, seems reasonable. Do you have thoughts otherwise on info that shouldn't be redacted for now? I'm inclined to redact everything and just have them log the struct names as is done for {scram_credentials}, with a comment warning to be careful about making sensitive info visible.

Reply:
yeh for these particular structs i think it makes sense to just log the structure name or something like that.

Reply:
Done